docs/devel/networking.md

```diff
@@ -1,4 +1,14 @@
 # VMI Networking
+Before we explain how KubeVirt performs VM networking configuration, it is
+paramount we separate the pod networking configuration and the  VM networking
+configuration concepts. Pod networking is out of the scope of this document;
+k8s is responsible to set up pod networking according to its configuration, via
+CNI. It is safe to assume it will simply plug a configured network interface
+into the pod, and through it, the pod connects to the outside world.
+
+The relevant part for KubeVirt is VM networking configuration. That happens in
+KubeVirt, and is what will be casually referred to as `binding` throughout this
```

Comment:
What is "in KubeVirt". KubeVirt is something that is "doing", not a place. Does it make sense?

Reply:
Done